src/topNavigation/topNavigation.tsx

```diff
@@ -40,9 +41,28 @@ interface State {
 interface TopNavigationLinkProps {
   to: string;
   label: string;
+  default?: string;
   className?: string;
 }
 
+export const NaviLinks: TopNavigationLinkProps[] = [
```

Comment:
Seems like this should start with lowercase `naviLinks`, as it is not a class or a component